app/renderer/src/components/Inspector/LocatedElements.jsx

```diff
@@ -46,7 +48,8 @@ const LocatedElements = (props) => {
       {locatedElements.length === 0 && (
         <Space className={InspectorStyles.spaceContainer} direction="vertical" size="small">
           <Row>
-            <i>{t('couldNotFindAnyElements')}</i>
+            <i>{`${t('couldNotFindAnyElements')} with locator `}</i>
```

Comment:
Please update the original message in the resources instead. Changing its payload here would lead to broken localizations. The message itself should be changed to (pay attention to the format specifiers presence)

`The locator [{{locatorTestStrategy}} = {{locatorTestValue}}] did not match any elements`

Reply:
@mykola-mokhnach That makes sense. I have updated the message in the translation now.